css/activities.css

```diff
@@ -150,6 +150,28 @@ body:not(.dark) #helpfulSearchDiv {
   text-align: left;
 }
 
+.block-dropdown {
```

Comment:
Maybe a better name for this? Something that describes the what the dropdown will contain?